src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchSimplificationFixAllProvider.cs

```diff
@@ -40,6 +41,13 @@ protected virtual SyntaxNode GetNodeToSimplify(SyntaxNode root, SemanticModel mo
             return root.FindNode(diagnostic.Location.SourceSpan, findInsideTrivia: true);
         }
 
+        protected virtual Task<Document> AddSimplifyAnnotationsAsync(Document document, SyntaxNode nodeToSimplify, CancellationToken cancellationToken)
+        {
+            throw new NotImplementedException();
+        }
+
+        protected virtual bool NeedsParentFixup { get { return false; } }
```

Comment:
Both the implementations simply return true - is there a need for this property?